### PR TITLE
Kirill.sitcevoy/serialization

### PR DESCRIPTION
**Pull request by kievzenit**
I can't find SoapFormatter in System.Runtime.Serialization.Formatters.Soap namespace. Someone have the same problem as mine?

### CODE REVIEW

File: Serialization/Serialization/Helper.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        static Helper()
+        {
+            xmlFormatter = new XmlSerializer(typeof(Student));
+            binaryFormatter = new BinaryFormatter();
+            
+        }
+
+        public static void SerializeToXml(Student student, Stream stream)
+        {
+            using (stream)
```
Comment (Artya) [inline Serialization/Serialization/Helper.cs]: you creating stream somewhere not in this class but dispose you executing here by using, why?

Comment (Artya) [reply Serialization/Serialization/Helper.cs]: you have to use using only one time

File: Serialization/Serialization/Helper.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        private static BinaryFormatter binaryFormatter;
+
+        static Helper()
+        {
+            xmlFormatter = new XmlSerializer(typeof(Student));
+            binaryFormatter = new BinaryFormatter();
+            
+        }
+
+        public static void SerializeToXml(Student student, Stream stream)
```
Comment (Artya) [inline Serialization/Serialization/Helper.cs:21]: better pass here only file location address

File: Serialization/Serialization/Helper.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        }
+        public static Student DeserializeXml(Stream stream)
+        {
+            var student = xmlFormatter.Deserialize(stream) as Student;
+
+            return student;
+        }
+        public static void SerializeToBinary(Student student, Stream stream)
+        {
+            using (stream)
```
Comment (Artya) [inline Serialization/Serialization/Helper.cs]: same problem here with this using

**Review by Artya — CHANGES_REQUESTED** [review]

File: Serialization/Serialization/Helper.cs (hunk 1 above)
Comment (kievzenit) [reply Serialization/Serialization/Helper.cs]: My fault, fixed in a2b8f77f25c975083ebee2c718ad445e5576ff02

File: Serialization/Serialization/Helper.cs (hunk 3 above)
Comment (kievzenit) [reply Serialization/Serialization/Helper.cs]: Fixed in a2b8f77f25c975083ebee2c718ad445e5576ff02

File: Serialization/Serialization/Helper.cs (hunk 2 above)
Comment (kievzenit) [reply Serialization/Serialization/Helper.cs:21]: Why? What if user wants to store xml not a file, but to memory, or somewhere else?

**Pull request closed without merging**